src/index.ts

```diff
@@ -25,6 +26,14 @@ renderBoard(highlightedInitialBoard);
 
 (async () => {
   while (currentGameState.isRunning) {
+    if (checkIfDraw(currentGameState.turnNumber)) {
+      currentGameState = setGameAsDraw(currentGameState);
+    }
+    if (currentGameState.isDraw === true) {
+      renderDrawMessage();
+      return;
```

Comment:
Instead of returning here you could have used the `isRunning` boolean from the `gameState`, just like @adguernier did. This would make the code more consistent.

Reply:
I did that but wanted to escape from current loop and avoid executing code below that point.